README.md

```diff
@@ -277,13 +278,13 @@ realm.close();
 
 # Realm Dart SDK
 
-The Realm Dart package is `realm_dart`
+The Realm Dart package is [realm_dart](https://pub.dev/packages/realm_dart).
 
 ## Environment setup for Realm Dart
 
 * Supported platforms are Windows, Mac and Linux.
 
-* Dart SDK ^2.17
+* Dart SDK ^2.17.5
```

Comment:
To be aligned to [Update to analyzer 5+](https://github.com/realm/realm-dart/pull/1072/)

Reply:
I don't understand the comment. You want to block this PR until the other PR lands ?

Reply:
I don't want to block this PR. I've just added a marker in case the other PR goes first.

Reply:
Sorry, @blagoev, I deleted your comment by mistake. It was duplicated somehow.
The comment was: "I don't understand the comment. You want to block this PR until the other PR lands ?"